include/OpenSpaceToolkit/Astrodynamics/Flight/System/PropulsionSystem.hpp

```diff
@@ -0,0 +1,144 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Flight_System_PropulsionSystem__
+#define __OpenSpaceToolkit_Astrodynamics_Flight_System_PropulsionSystem__
+
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+
+#include <OpenSpaceToolkit/Physics/Data/Direction.hpp>
+#include <OpenSpaceToolkit/Physics/Data/Scalar.hpp>
+#include <OpenSpaceToolkit/Physics/Unit.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Time.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace flight
+{
+namespace system
+{
+
+using ostk::core::types::Real;
+
+using ostk::physics::data::Direction;
+using ostk::physics::data::Scalar;
+using ostk::physics::units::Length;
+using ostk::physics::units::Mass;
+using ostk::physics::units::Time;
+using ostk::physics::units::ElectricCurrent;
+using ostk::physics::units::Angle;
+using ostk::physics::units::Derived;
+using ostk::physics::Unit;
+
+/// @brief                      Define a propulsion system (constant thrust, constant Isp for now)
+
+class PropulsionSystem
+{
+   public:
+    static Unit thrustSIUnit_;
+    static Unit specificImpulseSIUnit_;
+    static Unit massFlowRateSIUnit_;  // TBI: Define in ostk physics as proper units
```

Comment:
```suggestion
    static Unit thrustSIUnit;
    static Unit specificImpulseSIUnit;
    static Unit massFlowRateSIUnit;  // TBI: Define in ostk physics as proper units
```
Nit: Maybe make it without the underscore since it's public?

Reply:
Yup!